src/njoy21/input/DTFR/Card4/HollerithNames/test/HollerithNames.test.cpp

```diff
@@ -0,0 +1,58 @@
+#define CATCH_CONFIG_MAIN
+#include "catch.hpp"
+#include "njoy21.hpp"
+
+using namespace njoy::njoy21::input;
+
+SCENARIO( "HollerithNames input values",
+         "[Card4], [HollerithNames]" ){
+  Argument< DTFR::Card3::Iptotl > iptotl;
+  iptotl.value = 6 ;
+
+ GIVEN( "the correct number of names" ){
+
+    WHEN( "the names are of correct length" ){
+      iRecordStream< char> iss( std::istringstream("pheat1 edits dtfr") );
```

Comment:
The names have to be delimited by an apostrophe (`'`). All of the tests here have to have the apostrophe. We'll have to ask @apmccartney how to read them in that way.